conf/squid/common/rbd-3node-1client-rh.yaml

```diff
@@ -0,0 +1,38 @@
+globals:
```

Comment:
file name `conf/squid/common/rbd-3node-1client-rh.yaml` it's applicable to IBM as well
so why `rh` at the end can you please remove that like this `conf/squid/common/rbd-3node-1client.yaml`

Reply:
changed